practice_app/pages/api/getStatisticsModule.js

```diff
@@ -0,0 +1,16 @@
+exports.checking = function (val) {
+    let cnull = 'no'
+    let ansnull = 'yes'
+
```

Comment:
Please use better descriptive names, both for the function name and the variable names. I had to look through the code to understand that `cnull` means `countryIsNull`.

Reply:
done